chapters/03-funcoes.md

```diff
@@ -53,9 +53,9 @@ Os parâmetros de uma função comportam-se como variáveis regulares. Seu valor
 
 Uma propriedade importante das funções é que variáveis definidas dentro do corpo delas, incluindo seus parâmetros, são *locais* à própria função. Isso significa, por exemplo, que a variável `result` no exemplo `power` será criada novamente toda vez que a função for invocada, sendo que as diferentes execuções não interferem umas nas outras.
 
-Esse “senso de localidade” das variáveis se aplica somente aos parâmetros e às variáveis que forem declaradas usando a palavra-chave `var` dentro do corpo da função. Variáveis declaradas fora do contexto de alguma função são chamadas de *globais* (não-locais), pois elas são visíveis em qualquer parte da aplicação. É possível acessar variáveis *globais* dentro de qualquer função, contanto que você não tenha declarado uma variável local com o mesmo nome.
+Essa restrição particular das variáveis se aplica somente aos parâmetros e às variáveis que forem declaradas usando a palavra-chave `var` dentro do corpo de uma função. Variáveis declaradas fora do contexto de alguma função são chamadas de *globais* (não-locais), pois elas são visíveis em qualquer parte da aplicação. É possível acessar variáveis *globais* dentro de qualquer função, contanto que você não tenha declarado uma variável local com o mesmo nome.
```

Comment:
No início da frase, poderíamos alterar para algo do tipo:
 
* `Essa característica de "localidade" particular das variáveis [...]`
* `Essa "localidade" particular das variáveis [...]`
* `Essa característica de "localidade" das variáveis [...]`
* Ou manter

Eu não consigo ver onde a palavra "restrição" reflete o sentido do trecho original. Qual seu ponto de vista nessa parte?